### PR TITLE
Post method for adding reference data and Bootstrap Toast for success and error

### PR DESCRIPTION
**Pull request by jonathan-osorio**

### CODE REVIEW

File: src/app/reference-data/reference-data.component.ts [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+  constructor(private modalService: NgbModal, private apiService: RealestateApiService) { }
+
   ngOnInit(): void {
-  this.apiService.getAllReferenceData().subscribe((referenceDataList$: any) => {
-    this.referenceDataList = referenceDataList$.typologiesList;
-  })
-}
-  
-  openModal(){
+    this.apiService.getAllReferenceData().subscribe((referenceDataList$: any) => {
```
Comment (Silva096) [inline src/app/reference-data/reference-data.component.ts:21]: You need to merge this branch with master please.

File: src/app/services/realestate-api.service.ts [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
-  constructor(private httpCLient: HttpClient) {}
+  constructor(private httpClient: HttpClient) { }
+
+  getAllReferenceData(): Observable<ReferenceDataResponseDto> {
+    return this.httpClient.get<ReferenceDataResponseDto>(this.baseUrl + 'ReferenceData')
+  }
 
-  getAllReferenceData() : Observable<RefrenceDataResponseDto> {
-    return this.httpCLient.get<RefrenceDataResponseDto>(this.baseUrl + 'ReferenceData')
+  addReferenceData(selectedRefDataModalType: string, addRefDataValueForm: any) {
```
Comment (Silva096) [inline src/app/services/realestate-api.service.ts:25]: create reference data service for this endpoints please.

File: src/app/modals/reference-data-modal/reference-data-modal.component.ts [hunk 3]
```diff
@@ -1,23 +1,39 @@
...
 @Component({
   selector: 'app-reference-data-modal',
   templateUrl: './reference-data-modal.component.html',
   styleUrl: './reference-data-modal.component.css'
 })
-export class ReferenceDataModalComponent implements OnInit{
+export class ReferenceDataModalComponent implements OnInit {
   form = referenceDataForm;
+  selectedRefDataModalType: string;
+  selectedRefDataModalDescription: string;
```
Comment (Silva096) [inline src/app/modals/reference-data-modal/reference-data-modal.component.ts:16]: this is not being used

File: src/app/toast/toast-model.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { ToastClassEnum } from "../enums/toast-class-enum";
+
```
Comment (Silva096) [inline src/app/toast/toast-model.ts:2]: extra new line isn't needed

File: src/app/services/toast.service.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { Injectable, TemplateRef } from '@angular/core';
+import { Toast } from '../toast/toast-model';
+import { ToastClassEnum } from '../enums/toast-class-enum';
+
```
Comment (Silva096) [inline src/app/services/toast.service.ts:4]: extra new lines (2) aren't needed

**Review by Silva096 — APPROVED** [review]

**Review by RafaelMeha — APPROVED** [review]

**Pull request merged**